pkg/router/template/router.go

```diff
@@ -301,6 +302,17 @@ func genCertificateHostName(hostname string, wildcard bool) string {
 	return hostname
 }
 
+func randomEndpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit, random bool) []Endpoint {
```

Comment:
Can we doc the random argument (or perhaps call it shuffle?)

As something to consider, perhaps we should make it a string?  And support types "sorted" as the default, and "shuffled"... but that would allow other types in the future a little more easily.  Not that I can think of any off-hand.